packages/q/quickcpplib/xmake.lua

```diff
@@ -0,0 +1,23 @@
+package("quickcpplib")
+    set_kind("library")
+    set_homepage("https://github.com/ned14/quickcpplib")
+    set_description("Eliminate all the tedious hassle when making state-of-the-art C++ 14 - 23 libraries!")
+    set_license("Apache 2")
+
+    add_urls("https://github.com/ned14/quickcpplib.git")
+    add_versions("20221116", "52163d5a198f1d0a2583e683f090778686f9f998")
+
+    add_deps("cmake")
+    on_install(function (package)
+        local configs = {}
+        table.insert(configs, "-DCMAKE_BUILD_TYPE=" .. (package:debug() and "Debug" or "Release"))
```

Comment:
table.insert(configs, "-DBUILD_SHARED_LIBS=" .. (package:config("shared") and "ON" or "OFF"))